packages/components/src/VirtualizedList/CellDatetime/CellDatetime.component.js

```diff
@@ -53,35 +72,22 @@ export class CellDatetimeComponent extends React.Component {
 	render() {
 		const { cellData, columnData, t } = this.props;
 		const computedValue = computeValue(cellData, columnData, t);
+		const tooltipLabel = getTooltipLabel(cellData, columnData);
 
 		const cell = (
 			<div className={classnames('cell-datetime-container', styles['cell-datetime-container'])}>
 				{computedValue}
 			</div>
 		);
 
-		if (columnData.mode === 'ago') {
-			let tooltipLabel = '';
-
-			if (columnData.timeZone) {
-				tooltipLabel = talendUtils.date.formatToTimeZone(cellData, columnData.pattern || DATE_TIME_FORMAT, {
-					timeZone: columnData.timeZone,
-				});
-			} else {
-				tooltipLabel = format(cellData, columnData.pattern || DATE_TIME_FORMAT);
-			}
-
-			return (
-				<TooltipTrigger
-					label={tooltipLabel}
-					tooltipPlacement={columnData.tooltipPlacement || 'bottom'}
-				>
-					{cell}
-				</TooltipTrigger>
-			);
-		}
-
-		return cell;
+		return (
```

Comment:
you force the tooltip even if no label. I would prefer to keep it optional

Reply:
So it was decided by UX team to use tooltips for every column, cause when column is too small to show all the info, we need to show the tooltip https://jira.talendforge.org/browse/TDOPS-113